From: Matthias Badaire <mbadaire@mandrakesoft.com>
Subject: DLINK USB ethernet
To: jquintela@mandrakesoft.com, kernel@mandrakesoft.com
Date: 11 Jul 2001 18:47:59 +0200

Juan 
Could you add this patch to support the DU-E100 usb ethernet card ?

This should enable the automatic detection of the card by usbd and the
install

Thanks

--- linux/drivers/usb/pegasus.h.badz	Wed Jul 11 18:21:40 2001
+++ linux/drivers/usb/pegasus.h	Wed Jul 11 18:39:29 2001
@@ -151,6 +151,8 @@
 		DEFAULT_GPIO_RESET | PEGASUS_II | HAS_HOME_PNA )
 PEGASUS_DEV( "USB HPNA/Ethernet", VENDOR_ABOCOM, 0x4104,
 		DEFAULT_GPIO_RESET | HAS_HOME_PNA )
+PEGASUS_DEV( "USB HPNA/Ethernet", VENDOR_ABOCOM, 0x4002,
+		DEFAULT_GPIO_RESET | HAS_HOME_PNA )
 PEGASUS_DEV( "USB HPNA/Ethernet", VENDOR_ABOCOM, 0x4004,
 		DEFAULT_GPIO_RESET | HAS_HOME_PNA )
 PEGASUS_DEV( "USB HPNA/Ethernet", VENDOR_ABOCOM, 0x4007,

-- 
Matthias Badaire

